Update script.py

windowsx65 opened the pull request:
I have changed the display of the 'Key' and 'Button activated' values so that they are shown first and can be seen from the notification preview with the in-game overlay or smartphone notifications

windowsx65 committed:
Update script.py

I have changed the display of the 'Key' and 'Button activated' values so that they are shown first and can be seen from the notification preview with the in-game overlay or smartphone notifications

carlop3333 reviewed (changes requested):
wrong part (you need to edit from line 71 to 76)

windowsx65 commented:
> wrong part (you need to edit from line 71 to 76)

Thanks, I totally didn't know that. I send a new request.

windowsx65 committed:
Update script.py

I have changed the display of the 'Key' and 'Button activated' values so that they are shown first and can be seen from the notification preview with the in-game overlay or smartphone notifications and fixed my last mistake

carlop3333 reviewed (approved):
now it looks nice. ok
